include/rmm/cuda_stream_view.hpp

```diff
@@ -58,16 +70,22 @@ class cuda_stream_view {
 
   /**
    * @brief Implicit conversion to cudaStream_t.
+   *
+   * @return cudaStream_t The wrapped stream.
```

Comment:
I no longer think of stream_view as a wrapper. It's a non-owning reference.

```suggestion
   * @return cudaStream_t The underlying stream referenced by this cuda_stream_view
```